product_docs/docs/edbcloud/beta/administering_cluster/01_user_access.mdx

```diff
@@ -0,0 +1,100 @@
+
+EDB Cloud supports role based access control policies. A user with the **owner** role may assign roles to other users in the same organization. A user with the **owner** role may also define their own sets of roles which can be applied across the organization. 
+
+
+## Roles
+Access to EDB Cloud is controlled by roles. Roles are sets of permissions. You use roles to manage permissions assigned to users.
+
+### Default Roles
+
+* The predefined _owner_ role created during marketplace sign up has all permissions.
+* Each organization will have two additional default roles of _reader_ and _contributor_ available during Preview.
+* These roles can be edited by changing their name, description, or permissions.
+
+### Role Permissions
+
+Permissions are generally represented in the format _action:object_ where _action_ represents an operation that can be performed and _object_ represents a category of EDB Cloud functionality.
+
+The available _actions_ are:
+* _create _
+* _read  _
+* _update _
+* _delete_
+
+The available _objects_ are:
+* _databases_
+* _backups_
+* _events_
+* _permissions_
+* _roles_
+* _users_
+* _billing_
+* _versions_
+
+
+### Editing Roles
+
+To edit roles:
+1. Navigate to Admin > Roles.
+2. Select the edit icon for the role in the list.
+
+#### Changing Role Name
+
+To change the name or dscription of the role:
+1. Select the **Settings** tab.
+2. Edit **Name** or **Description**.
+3. Press **Save**.
+
+#### Changing Role Permissions
+
+To change permissions associated with the role:
+1. Select **Permissions** tab.
+2. Select the **Change Permissions** button on the top right.
+3. Select the list of permissions you want to be associated with the role.
+4. Press **Submit**
+
+## Users
+
+When you configured your Azure subscription, you also enabled EDB Cloud to authenticate users from your organization using Azure Active Directory (AD). Before users become visible in the EDB Cloud **Users** screen they need to sign in to EDB Cloud using special emails designated by your organization to sign in using Azure AD. New users signed in to EDB Cloud have a minimum set of permissions until you assign them a role.
+
+### Assigning Roles to Users
+To assign appropriate roles to users:
+1. Navigate to Admin > Users.
+2. Select the edit icon for the user.
+3. Select **Assign Roles**.
+4. Select or deselect roles for the user.
+5. Select **Submit**.
+
+!!! Note
+    For a user's role assignment to take effect, the user needs to log out from EDB Cloud and log in again.
+
+### Viewing Users
+
+To view all users from your organization that have logged in at least once:
+
+1. Navigate to Admin > Users.
+2. View the list of users sorted by most recent log in.
+
+## Example Scenario
+1. AcmeBank owns a subcription to Azure and the acmebank.com Azure AD.
+1. AcmeBank purchases a subscription to the EDB Cloud and assigns the name acmebank-com for their organization.
```

Comment:
Can we avoid using "subscription" here? It's a technical term in Azure and muddies the waters.

Reply:
just for the second sentence or both?